src/api.js

```diff
@@ -5,10 +5,27 @@ const extractRawData = async (location) => {
     // fetch data from api
     const response = await fetch(apiUrl);
 
+    // parse promise data into json
+    // const data = await response.json();
```

Comment:
Kindly remove commented code

Reply:
Issue solved. Thank you for your feedback